source/jormungandr/jormungandr/interfaces/v1/serializer/status.py

```diff
@@ -204,6 +204,12 @@ def get_arrival_radius(self, obj):
         return obj.get('arrival_radius')
 
 
+class BSSStationsSerializer(OutsideServiceCommon):
```

Comment:
Minor for accuracy:
```suggestion
class BSSStationsServiceSerializer(OutsideServiceCommon):
```
or Provider if it suits you more.

Reply:
:+1: